scripts/aws/monitor.py

```diff
@@ -269,7 +281,8 @@ def add_cloudwatchlog_role_to_apigateway(client, role_arn):
                      get_topic_arn(sns_client, 'aws-non-critical-alert'))
 
     lambda_function_name = 'cloudwatch-logs-splunk'
-    lambda_exec_role_arn = create_role_with_inline_policy(iam_client, 'lambda_basic_execution_monitor_cloudwatch_logs',
+    lambda_role_name = 'lambda_basic_execution_monitor_cloudwatch_logs'
```

Comment:
Is one role that is share by all environments acceptable in this case?  If so, that's fine, but lets pull the role name up into a global variable.